backend/src/main/java/com/benine/backend/database/Database.java

```diff
@@ -22,40 +22,25 @@
 
   /**
    * Add a preset to the database.
-   *
-   * @param camera             ID of the camera
    * @param preset             The preset to be added
    * @throws SQLException No right connection found
    */
-  void addPreset(int camera, Preset preset) throws SQLException;
+  void addPreset(Preset preset) throws SQLException;
 
   /**
    * Delete a preset from the database.
    * @param camera ID of the camera
-   * @param cameraPresetNumber ID of the preset for the camera
    * @throws SQLException No right connection found
    */
-  void deletePreset(int camera, int cameraPresetNumber) throws SQLException;
+  void deletePreset(int camera) throws SQLException;
```

Comment:
That would indeed make more sense as for a camera can have multiple presets so now it is not clear which preset you are deleting?